modules/ob-release-notes-1-5.adoc

```diff
@@ -0,0 +1,65 @@
+// This module is included in the following assemblies:
+// * about/ob-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ob-release-notes-1-5_{context}"]
+= Release notes for {builds-shortname} 1.5
+
+{builds-shortname} 1.5 is now available on {ocp-product-title} 4.16, 4.17, 4.18, and 4.19.
+
+[id="new-features-1-5_{context}"]
+== New features
+
+The following sections highlight what is new in {builds-shortname} 1.5.
+
+* With this update, {builds-operator} deploys Shipwright v0.16 components.
+* With this update, you can use the optional `.spec.nodeSelector` value with `shp` CLI to specify which nodes build run pods should run on.
```

Comment:
With this update, you can use the optional .spec.nodeSelector field with the shp CLI to specify the nodes on which build run pods should run.